docs/reference/commandline/login.md

```diff
@@ -162,7 +164,7 @@ designated programs to handle credentials for *specific registries*. The default
 credential store (`credsStore` or the config file itself) will not be used for
 operations concerning credentials of the specified registries.
 
-### Logging out
```

Comment:
Good catch; would be worth to add a "Related commands" at the end of this file, to link to `logout` (similar to https://github.com/docker/cli/blob/master/docs/reference/commandline/plugin_create.md#related-commands)

Reply:
done